src/wasm-link.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2016 WebAssembly Community Group participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef WASM_LINK_H_
+#define WASM_LINK_H_
+
+#include "binary.h"
+#include "common.h"
+#include "vector.h"
+
+#define LINKER_IMPORT_MODULE_NAME "linker"
```

Comment:
Done.  Although do you really want me to change all the types in the this file.  It would be quite a big change :(